packages/manager/apps/license/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@ovh-ux/manager-license-app",
+  "version": "0.0.1",
+  "description": "OVHcloud license management app",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/license"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "webpack --env.production",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-license-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-license-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-license-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
+    "@ovh-ux/manager-core": "^12.0.0 || ^13.0.0",
+    "@ovh-ux/manager-license": "^0.0.1 || ^1.0.0",
+    "@ovh-ux/manager-preloader": "^1.0.0",
+    "@ovh-ux/ng-ovh-payment-method": "^8.1.2",
+    "@ovh-ux/ng-ovh-request-tagger": "^1.1.0",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.5.1",
+    "@ovh-ux/ng-ovh-swimming-poll": "^5.0.3",
+    "@ovh-ux/ufrontend": "^1.1.3 || ^2.0.0",
+    "@ovh-ux/ui-kit": "^4.7.0",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "1.7.x",
+    "angular-ui-bootstrap": "1.3.3",
+    "bootstrap": "~3.3.7",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "font-awesome": "~4.7.0",
+    "ipaddr.js": "^1.6.0",
+    "jsurl": "^0.1.4",
+    "lodash-es": "^4.17.15",
+    "moment": "^2.24.0",
+    "oclazyload": "^1.1.0",
+    "office-ui-fabric-core": "^11.0.0",
+    "ovh-ui-kit-bs": "^4.2.0",
+    "regenerator-runtime": "^0.13.7",
+    "urijs": "^1.19.1",
+    "whatwg-fetch": "^3.5.0"
+  },
+  "devDependencies": {
+    "@ovh-ux/manager-webpack-config": "^3.1.1 || ^4.0.0",
+    "eslint-plugin-angular": "^4.0.1",
```

Comment:
Seems not by default in https://github.com/ovh/manager/tree/develop/packages/manager/tools/sao-ovh-manager-app